fhir-server/src/test/java/com/ibm/fhir/server/test/BundleValidationTest.java

```diff
@@ -0,0 +1,692 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.ibm.fhir.server.test;
+
+import static com.ibm.fhir.model.type.String.string;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.fail;
+
+import java.util.List;
+
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import com.ibm.fhir.config.FHIRConfiguration;
+import com.ibm.fhir.config.FHIRRequestContext;
+import com.ibm.fhir.core.HTTPReturnPreference;
+import com.ibm.fhir.exception.FHIRException;
+import com.ibm.fhir.exception.FHIROperationException;
+import com.ibm.fhir.model.resource.Bundle;
+import com.ibm.fhir.model.resource.Bundle.Entry.Response;
+import com.ibm.fhir.model.resource.Bundle.Entry.Search;
+import com.ibm.fhir.model.resource.Condition;
+import com.ibm.fhir.model.resource.OperationOutcome;
+import com.ibm.fhir.model.resource.OperationOutcome.Issue;
+import com.ibm.fhir.model.type.CodeableConcept;
+import com.ibm.fhir.model.type.Id;
+import com.ibm.fhir.model.type.Meta;
+import com.ibm.fhir.model.type.Narrative;
+import com.ibm.fhir.model.type.Reference;
+import com.ibm.fhir.model.type.UnsignedInt;
+import com.ibm.fhir.model.type.Uri;
+import com.ibm.fhir.model.type.Xhtml;
+import com.ibm.fhir.model.type.code.BundleType;
+import com.ibm.fhir.model.type.code.HTTPVerb;
+import com.ibm.fhir.model.type.code.IssueSeverity;
+import com.ibm.fhir.model.type.code.IssueType;
+import com.ibm.fhir.model.type.code.NarrativeStatus;
+import com.ibm.fhir.model.type.code.SearchEntryMode;
+import com.ibm.fhir.persistence.FHIRPersistence;
+import com.ibm.fhir.registry.FHIRRegistry;
+import com.ibm.fhir.server.util.FHIRRestHelper;
+
+public class BundleValidationTest {
+
+    FHIRPersistence persistence;
+    FHIRRestHelper helper;
+
+    @BeforeClass
+    void setup() throws FHIRException {
+        FHIRConfiguration.setConfigHome("src/test/resources");
+        FHIRRequestContext.get().setTenantId("default");
```

Comment:
is this needed for the tests?

Reply:
not needed - removed unnecessary setup and teardown code